t/lib/t/MusicBrainz/Server/WebService/Format.pm

```diff
@@ -0,0 +1,136 @@
+package t::MusicBrainz::Server::WebService::Format;
+
+use JSON qw( decode_json encode_json );
+use Test::JSON import => [qw( is_valid_json is_json )];
+use Test::More;
+use Test::Routine;
+use Test::XML;
```

Comment:
Hm, `Test::XML` isn't in the cpanfile, so the tests are failing. We do have `Test::XML::SemanticCompare`. Is this module needed?